css/index.css

```diff
@@ -1 +1,229 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+    font-size:62.5%;
+    box-sizing: content-box;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+a {
+	text-decoration: none;
+}
+.button{
+	width: 150px;
+	height:20px;
+	margin: 0 auto;
+}
+.spacer{
+	margin-top:20px;
+	margin-bottom:20px;
+	height:100%;
+	width:100%;
+}
+.header {
+    font-family:Patua One;
+    font-size:2rem;
+	font-weight: bold;
+	margin-top:10px;
+	margin-bottom:10px;
+}
+.copyRight{
+	text-align: center;
+	margin-top:10px;
+	margin-bottom: 100px;
+}
+.textBlocks{
+    font-size:1.6rem;
+}
+.outerContainer {
+	display:flex;
+	max-width:880px;
+	margin:0 auto;
+	flex-direction: column;
+}
+.outerContainer .mainNav{
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	width:100%;
+}
+.outerContainer .mainNav .nav a {
+	padding:20px;
```

Comment:
if you remove this at mobile-view, it will allow the nav items to reach the end of the scree like the design file.